terraform/testcases/standard_otlp_metric_trace/parameters.tfvars

```diff
@@ -0,0 +1,6 @@
+validation_config = "standard-otel-trace-metric-validation.yml"
+
+soaking_data_mode = "trace"
+
+# Replace with ecr standard go sample app image when available
+sample_app_image = ""
```

Comment:
Do we expect the images to be available on ECR or can we build them locally in the test environment?

Reply:
Also, this standard template should be used with a variety of sample apps right? Should this value be set at test runtime and not explicitly defined in the parameters file?

Reply:
We can explore both options when it comes to ECR vs. building images locally, and I can edit this behavior depending on what we decide.  If the images were available on ECR, I believe it would be easier to just pull the different images in the workflows and allow people to easily grab the sample app images if needed.  However, it would require more maintenance in ECR and takes up resources.  If the images were built locally in the test environment then that maintenance would not be required and resources are saved, but the workflow would require more work to build each different image and people would have to learn how to build each respective image on their own if needed.  What do you think would be the best path considering this?

Reply:
The original idea was to pull the images from ECR where they would be maintained. But it makes sense to also build them locally in order to know that an image passes all testing to then being updated or uploaded to ECR.